www/docs/en/6.x/config_ref/images.md

```diff
@@ -119,6 +120,39 @@ different screen resolutions.
 - [App icon and image size guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html)
 
 ##Windows
+
+For Windows the recommended approach to define app icons is to use `target` attribute.
+
+```xml
+    <platform name="windows">
+        <icon src="res/windows/storelogo.png" target="StoreLogo" />
+        <icon src="res/windows/smalllogo.png" target="Square30x30Logo" />
+        <icon src="res/Windows/Square44x44Logo.png" target="Square44x44Logo" />
```

Comment:
I'm confused - does the `target` name have to be this? How does this eventually get represented in the manifest - does the target/src filename not matter?